src/main/java/trackr/storage/JsonAdaptedTask.java

```diff
@@ -0,0 +1,79 @@
+package trackr.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import trackr.commons.exceptions.IllegalValueException;
+import trackr.model.task.Task;
+import trackr.model.task.TaskDeadline;
+import trackr.model.task.TaskName;
+import trackr.model.task.TaskStatus;
+
+/**
+ * Jackson-friendly version of {@link Task}.
+ */
+class JsonAdaptedTask {
+
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Task's %s field is missing!";
+
+    private final String name;
+    private final String deadline;
+    private final String status;
+
+    /**
+     * Constructs a {@code JsonAdaptedPerson} with the given person details.
+     */
+    @JsonCreator
+    public JsonAdaptedTask(@JsonProperty("name") String name, @JsonProperty("deadline") String deadline,
+                             @JsonProperty("status") String status) {
+        this.name = name;
+        this.deadline = deadline;
+        this.status = status;
```

Comment:
Can give a more meaningful name to these fields like taskStatus instead of just status